arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -0,0 +1,691 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CognitiveServicesManagementClient",
+    "description": "Cognitive Services Management Client",
+    "version": "2016-02-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{accountName}": {
+      "put": {
+        "tags": [
+          "CognitiveServicesAccounts"
+        ],
+        "description": "Accounts is a resource group wide resource type. It holds the keys for developer to access intelligent APIs. It's also the resource type for billing.",
```

Comment:
This description does not describe the operation. It should start with "Creates a CognitiveServices Account. Acounts..."